data_structures/src/chain.rs

```diff
@@ -53,6 +53,9 @@ pub struct ChainInfo {
 
     /// Checkpoint of the last block in the blockchain
     pub highest_block_checkpoint: CheckpointBeacon,
+
+    /// Checkpoint of the last vrf otuput in the blockchain
```

Comment:
This comment is a bit confusing

```suggestion
    /// Checkpoint and VRF hash of the highest block in the blockchain
```

Reply:
`block chain` indeed :stuck_out_tongue: